sessionize/lib/src/main/java/co/touchlab/sessionize/platform/NotificationsJVM.kt

```diff
@@ -17,7 +17,7 @@ import android.content.ComponentName
 
 val notificationPublisher: BroadcastReceiver = NotificationPublisher()
 
-actual fun createLocalNotification(title:String, message:String, timeInMS:Long, notificationId: Int) {
+actual fun createLocalNotificationOnPlatform(title:String, message:String, timeInMS:Long, notificationId: Int) {
```

Comment:
This class seems slightly incorrectly named because it has the Android implementation, not just a JVM one

Reply:
What would you recommend? This is also in the iOS implementation so it should sound multi-platform.